parameters/examples/create_genesis_block.rs

```diff
@@ -50,7 +50,7 @@ pub fn generate<C: BaseDPCComponents>() -> Result<Vec<u8>, TransactionError> {
         difficulty_target: 0x07FF_FFFF_FFFF_FFFF_u64,
         nonce: 0,
         pedersen_merkle_root_hash: PedersenMerkleRootHash([0u8; 32]),
-        proof: ProofOfSuccinctWork::default(),
+        proof: ProofOfSuccinctWork([0; 972]),
```

Comment:
The `Default` impl is no longer available, so I just hardcoded the value it used to provide.

Reply:
This is correct, thank you for calling that out.